pipelineutilities/pipelineutilities/search_files.py

```diff
@@ -104,8 +104,8 @@
     ],
     "ArchivesSpace": [
         r"([a-zA-Z]{3}_[0-9]{9})",
+        r"([a-zA-Z]{3}-[a-zA-Z]{3}_[0-9]{3,4}-[0-9]{3})",
```

Comment:
Tbh, these patterns still may be unnecessarily picky. The important part is the dash and underscore delimiters; any reason to restrict it to 3 or 4 characters in each part of the path?